src/IECoreHoudini/GEO_CobIOTranslator.cpp

```diff
@@ -163,3 +164,27 @@ GA_Detail::IOStatus GEO_CobIOTranslator::fileSaveToFile( const GEO_Detail *geo,
 	
 	return fileSaveToFile( geo, fileName );
 }
+
+bool GEO_CobIOTranslator::fileStat( const char *fileName, GA_Stat &stat, uint level )
+{
+	try
+	{
+		ReaderPtr reader = Reader::create( fileName );
+		if ( !reader )
+		{
+			return false;
+		}
+		
+		CompoundObjectPtr header = reader->readHeader();
+
+		UT_BoundingBox bbox = convert<UT_BoundingBox>( reader->readHeader()->member<const Box3fData>( "bound", true )->readable() );
```

Comment:
I think rather than using the `true` argument for create if missing, you should return `false` when its missing. Otherwise you're returning a default initialized `Box3f`